docs/zh/query_language/create.md

```diff
@@ -1 +1,156 @@
-../../en/query_language/create.md
\ No newline at end of file
+## CREATE DATABASE
+
+创建 'db_name' 数据库.
+
+```sql
+CREATE DATABASE [IF NOT EXISTS] db_name
+```
+
+一个数据库是表的一个目录. 如果包含'IF NOT EXISTS', 如果数据库已经存在，则查询不返回错误.
+
+<a name="query_language-queries-create_table"></a>
+
+## CREATE TABLE
+
+'CREATE TABLE' 语句有几种形式.
```

Comment:
We should use Chinese symbols in zh docs, eg `，` ,  `。`

Reply:
When original English docs is `CREATE TABLE` , we should keep the back quote in ZH docs instead of using  'CREATE TABLE'  because it will have different styles in htmls during markdown parsing.

Reply:
take it